### PR TITLE
Dev ambr 124

### PR DESCRIPTION
**Pull request by msingh05**

### CODE REVIEW

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+  @Bean
+  public ArticleListCrudService articleListCrudService() {
+    mockArticleListCrudService = spy(ArticleListCrudServiceImpl.class);
+    LOG.debug("articleListCrudService() * --> {}", mockArticleListCrudService);
+    return mockArticleListCrudService;
+  }
+
+  @Bean
+  public ArticleCrudService articleCrudService() {
```
Comment (sjtower) [inline src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: this bean is already declared in `AbstractStubbingArticleTest`

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+    newArticleIds.add(ArticleIdentifier.create(article.getDoi()));
+
+    ArticleListView newArticleListView = mockArticleListCrudService.update(identity, Optional.empty(), Optional.ofNullable(newArticleIds));
+    assertThat(newArticleListView.getIdentity()).isEqualTo(identity);
+    assertThat(newArticleListView.getArticleList().getArticles().size()).isEqualTo(newArticleIds.size());
+  }
+
+  @Test
+  public void testUpdate_SucceedForRemoveArticlesList() {
+    ArticleListIdentity identity = createStubArticleListIdentity();
```
Comment (sjtower) [inline src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: This line is common in a lot of tests, you should consider moving it to a common init() method.

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+            "valid-journal", articleList
+        })));
+
+    ServiceResponse<ArticleListView> response = mockArticleListCrudService.read(identity);
+
+    ArticleListView newArticleListView = response.getBody();
+    assertThat(newArticleListView.getIdentity()).isEqualTo(identity);
+    assertThat(newArticleListView.getArticleList()).isEqualTo(articleList);
+
+//    // commented out code is needed to test without modifying ServiceResponse.java
```
Comment (sjtower) [inline src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: Lets resolve this - I don't like the idea of committing commented-out code. Modifying the source is fine in my opinion.

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java (hunk 1 above)
Comment (msingh05) [reply src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: removed.

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java (hunk 2 above)
Comment (msingh05) [reply src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: If it is in init() (or someother) method, then identity and articleListView will need to be defined as member variables. So will make it harder to read within the test function.I would like to leave it as it is.

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java (hunk 3 above)
Comment (msingh05) [reply src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: removed

File: src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+//    // and also needs SampleSerializer and localGson later.
+//    ResponseEntity<?> entity = response.asJsonResponse(localGson());
+//    String body = new String((byte[]) entity.getBody(), Charset.forName("UTF8"));
+//    logger.debug(body);
+  }
+
+//  public static class SampleSerializer implements JsonSerializer<ArticleRevisionView.Factory>
```
[the remark this answers is not in the document]
Comment (msingh05) [reply src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java]: removed

**Comment by msingh05** [comment]
@randy-tano pushed the new code. please review it again.

**Pull request merged**